### PR TITLE
Feature/flurry

### PR DESCRIPTION
**Pull request by dpsm**

### CODE REVIEW

File: wings-gcp/src/main/java/com/groundupworks/wings/gcp/GoogleCloudPrintEndpoint.java [hunk 1]
```diff
@@ -206,9 +215,27 @@ public LinkInfo getLinkInfo() {
...
+                                parameters.put("message", gcpResponse.message);
+                                if (gcpResponse.hasSucceeded) {
+                                    mDatabase.markSuccessful(shareRequest.getId());
+                                    shareCount++;
+                                    sLogger.log("gcp_queue_success", parameters);
+                                } else {
+                                    mDatabase.markFailed(shareRequest.getId());
+                                    sLogger.log("gcp_queue_failed", parameters);
+                                }
+                            } catch (IOException e) {
```
Comment (benhylau) [inline wings-gcp/src/main/java/com/groundupworks/wings/gcp/GoogleCloudPrintEndpoint.java]: `markFailed()` here?

Comment (dpsm) [inline wings-gcp/src/main/java/com/groundupworks/wings/gcp/GoogleCloudPrintEndpoint.java]: Why not?

Comment (benhylau) [inline wings-gcp/src/main/java/com/groundupworks/wings/gcp/GoogleCloudPrintEndpoint.java]: It wasn't there. I added it. That's correct right?

Comment (dpsm) [inline wings-gcp/src/main/java/com/groundupworks/wings/gcp/GoogleCloudPrintEndpoint.java]: Right

**Pull request merged**